src/libs/ReportUtils.js

```diff
@@ -151,14 +151,25 @@ function canEditReportAction(reportAction) {
 }
 
 /**
- * Can only delete if it's an ADDCOMMENT, the author is this user.
+ * Whether the Money Request report has been reimbursed
```

Comment:
Maybe `isSettled` isn't the best func name then? Can we tie the comment a bit closer to the function name?

Reply:
I'm a bit confused. Are you suggesting that we change the function name or the comment? 😅